### ISSUE TITLE
Fix docstring errors in batchnorm.py, activation.py

### ISSUE THREAD

**Issue by svekars** [pytorch/pytorch#112602]
Please fix the following issues. To test locally, run: 
 ```
pydocstyle path-to-file --count
```
Run this command before and after fixing the errors. This command prints out the number of errors at the end of the output. Paste the output to your PR description with the number before and after. 
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Threshold`, **Line**: 19, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `ReLU`, **Line**: 69, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `ReLU`, **Line**: 69, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `RReLU`, **Line**: 111, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `RReLU`, **Line**: 111, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `RReLU`, **Line**: 111, **Description**: First line should end with a period (not ',')
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Hardtanh`, **Line**: 177, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `ReLU6`, **Line**: 246, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Sigmoid`, **Line**: 276, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Hardsigmoid`, **Line**: 300, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `SiLU`, **Line**: 363, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `SiLU`, **Line**: 363, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Mish`, **Line**: 404, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Mish`, **Line**: 404, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Hardswish`, **Line**: 440, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Hardswish`, **Line**: 440, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Hardswish`, **Line**: 440, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `ELU`, **Line**: 480, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `ELU`, **Line**: 480, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `ELU`, **Line**: 480, **Description**: First line should end with a period (not 'd')
- **File**: `torch/nn/modules/activation.py`, **Entity**: `CELU`, **Line**: 526, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `CELU`, **Line**: 526, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `SELU`, **Line**: 570, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `SELU`, **Line**: 570, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `GLU`, **Line**: 619, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `GLU`, **Line**: 619, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `GLU`, **Line**: 619, **Description**: First line should end with a period (not 'n')
- **File**: `torch/nn/modules/activation.py`, **Entity**: `GELU`, **Line**: 652, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `GELU`, **Line**: 652, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Hardshrink`, **Line**: 693, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `LeakyReLU`, **Line**: 735, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `LeakyReLU`, **Line**: 735, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `LogSigmoid`, **Line**: 786, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softplus`, **Line**: 809, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softplus`, **Line**: 809, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softplus`, **Line**: 809, **Description**: First line should end with a period (not '*')
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softshrink`, **Line**: 851, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softshrink`, **Line**: 851, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `MultiheadAttention`, **Line**: 909, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `MultiheadAttention`, **Line**: 909, **Description**: First line should end with a period (not 'n')
- **File**: `torch/nn/modules/activation.py`, **Entity**: `forward`, **Line**: 1050, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `forward`, **Line**: 1050, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `forward`, **Line**: 1050, **Description**: Docstring is under-indented
- **File**: `torch/nn/modules/activation.py`, **Entity**: `forward`, **Line**: 1050, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `merge_masks`, **Line**: 1250, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `merge_masks`, **Line**: 1250, **Description**: First line should end with a period (not 'k')
- **File**: `torch/nn/modules/activation.py`, **Entity**: `PReLU`, **Line**: 1291, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `PReLU`, **Line**: 1291, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softsign`, **Line**: 1357, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Tanhshrink`, **Line**: 1380, **Description**: First line should end with a period (not '
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softmin`, **Line**: 1403, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softmin`, **Line**: 1403, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softmin`, **Line**: 1403, **Description**: First line should end with a period (not 'r')
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softmax`, **Line**: 1450, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softmax`, **Line**: 1450, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `Softmax`, **Line**: 1450, **Description**: First line should end with a period (not 'r')
- **File**: `torch/nn/modules/activation.py`, **Entity**: `LogSoftmax`, **Line**: 1534, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `LogSoftmax`, **Line**: 1534, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/activation.py`, **Entity**: `LogSoftmax`, **Line**: 1534, **Description**: First line should end with a period (not 'l')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `_NormBase`, **Line**: 18, **Description**: First line should end with a period (not 'm')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `BatchNorm1d`, **Line**: 234, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `BatchNorm1d`, **Line**: 234, **Description**: First line should end with a period (not 'r')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `LazyBatchNorm1d`, **Line**: 307, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `LazyBatchNorm1d`, **Line**: 307, **Description**: First line should end with a period (not 'f')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `BatchNorm2d`, **Line**: 342, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `BatchNorm2d`, **Line**: 342, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `LazyBatchNorm2d`, **Line**: 417, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `LazyBatchNorm2d`, **Line**: 417, **Description**: First line should end with a period (not 'f')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `BatchNorm3d`, **Line**: 450, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `BatchNorm3d`, **Line**: 450, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `LazyBatchNorm3d`, **Line**: 523, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `LazyBatchNorm3d`, **Line**: 523, **Description**: First line should end with a period (not 'f')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `SyncBatchNorm`, **Line**: 556, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `SyncBatchNorm`, **Line**: 556, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `convert_sync_batchnorm`, **Line**: 771, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `convert_sync_batchnorm`, **Line**: 771, **Description**: First line should end with a period (not 'o')
- **File**: `torch/nn/modules/batchnorm.py`, **Entity**: `convert_sync_batchnorm`, **Line**: 771, **Description**: First line should be in imperative mood; try rephrasing (found 'Helper')

cc @carljparker

**Comment by Jarlaze** [comment]
Hi @svekars could you tell me where I can find linting settings? I have a hard time reproducing pydocstyle output. I have tried to install linting as suggested https://github.com/pytorch/pytorch/wiki/lintrunner and https://github.com/pytorch/pytorch/blob/fd209543d5dfced3146d64df3af6361fc61fb74a/CONTRIBUTING.md#local-linting without success.

**Comment by svekars** [comment]
You can also try to use the updated instructions from [the newer issues](https://github.com/pytorch/pytorch/issues/113194) and run `ruff`:
```
ruff --select RULECODE --fix path_to_file
```
